Frontend/models/game-board-model.js

```diff
@@ -1,7 +1,41 @@
 var oswebbleData = {
     selectedTileId: '',
     selectedTileParentId: '',
-    selectedSquareId: ''
+    selectedSquareId: '',
+    // can change to computed property
+    rows: generateTableRows(),
+    squares: generateSquares()
+}
+
+function generateTableRows () {
+    var row = 11;
```

Comment:
Please abstract these values into some config file like `constants.js` and do something such as `export const rows = 11` and `export const cols = 11` and then import the config file wherever you need those values